### PR TITLE
Speech recognition (Workshop bounty)

### PR DESCRIPTION
**Pull request by wollygfx**
@sampoder Did requested changes at https://hackclub.slack.com/files/USNPNJXNX/F01HASFTGUE/screen_recording_2020-12-23_at_7.05.02_pm.mov

### CODE REVIEW

**Comment by wollygfx** [comment]
Also, remember that the workshop only works for google chrome

File: workshops/Speech_recognition/README.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+}
+```
+Let's break this down:
+- The `background-color`, `color` and `font-size` properties give the buttons a background color, a color to the text and a size to the text respectively.
+- The `padding` and `margin` properties are part of the [CSS box model](https://www.w3schools.com/css/css_boxmodel.asp).
+- The `border` and `border-radius` allow you to specify the style, width, and color of an element's border and define the radius of the element's corners respectively.
+- The `cursor` property sets the type of mouse cursor, if any, to show when the mouse pointer is over the buttons.
+- The `overflow-y` property sets what shows when content overflows a block-level element's top and bottom edges. This may be nothing, a scroll bar, or the overflow content.
+
+Finally, this is how our website looks like:
```
Comment (sampoder) [inline workshops/Speech_recognition/README.md:107]: Could we not use finally twice so quickly?

File: workshops/Speech_recognition/README.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+## Hack it
+Yay! you made it to the end of this workshop. I know this was a pretty simple workshop, but you can still make pretty complex stuff... Let me give you some ideas:
+- Use [if statatements](https://www.w3schools.com/js/js_if_else.asp) to do certaing thing when the speech recognition app returns text.
+- Create an app that play, stop and skip songs.
+- Style your app to make it look cool.
+
+## Live demos
+Here are some demos made by other people:
+- [An app that takes you to certaing link depending on the word said.](https://codepen.io/Rumyra/pen/bCphe)
+- [Phrase matcher app](https://mdn.github.io/web-speech-api/phrase-matcher/)
```
Comment (sampoder) [inline workshops/Speech_recognition/README.md:215]: Could you give a bit more context here

**Review by sampoder — APPROVED** [review]
🎉 Other than the above, this is pretty good!

**Comment by MatthewStanciu** [comment]
yay!

**Pull request merged**